.eslintrc.json

```diff
@@ -27,7 +26,7 @@
     "@typescript-eslint/func-call-spacing": ["error"],
     "@typescript-eslint/member-ordering": ["warn"],
     "@typescript-eslint/no-require-imports": ["error"],
-    "react/no-unused-prop-types": ["warn", { "skipShapeProps": true }],
+    "react/no-unused-prop-types": "off",
```

Comment:
Why was this turned off?

Reply:
there were a lot of those errors in some 3rd party plugin usages if I remember right. My idea was to make a minimum upgrade to get the SCW-Jest to work. I think this could have been fixed by upgrading rest of the packages as well.

Reply:
If I added the `"react/no-unused-prop-types": ["warn", { "skipShapeProps": true }],` back now I got:
```typescript
palvelutarjotin-ui> yarn lint
yarn run v1.22.19
$ eslint --ext js,ts,tsx src

palvelutarjotin-ui\src\common\components\datepicker\Datepicker.tsx
  38:3  warning  'disabled' PropType is defined but prop is never used  react/no-unused-prop-types

palvelutarjotin-ui\src\common\components\datepicker\TimesList.tsx
  11:3  warning  'times' PropType is defined but prop is never used        react/no-unused-prop-types
  12:3  warning  'onTimeClick' PropType is defined but prop is never used  react/no-unused-prop-types
  13:3  warning  'datetime' PropType is defined but prop is never used     react/no-unused-prop-types

palvelutarjotin-ui\src\domain\event\occurrences\EnrolmentButtonCell.tsx
  21:3  warning  'event' PropType is defined but prop is never used                react/no-unused-prop-types
  22:3  warning  'value' PropType is defined but prop is never used                react/no-unused-prop-types
  23:3  warning  'selectedOccurrences' PropType is defined but prop is never used  react/no-unused-prop-types
  24:3  warning  'enrolOccurrence' PropType is defined but prop is never used      react/no-unused-prop-types
  25:3  warning  'selectOccurrence' PropType is defined but prop is never used     react/no-unused-prop-types
  26:3  warning  'deselectOccurrence' PropType is defined but prop is never used   react/no-unused-prop-types
  27:3  warning  'neededOccurrences' PropType is defined but prop is never used    react/no-unused-prop-types

✖ 11 problems (0 errors, 11 warnings)
```

Thinking whether those could easily be tackled in some other way...

Reply:
- Datepicker doesn't support `disabled` at all so that property could be removed
- TimesList itself does use all of those props (times, onTimeClick and datetime), that might be [false positive from ESLint](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unused-prop-types.md#known-issueslimitations). Let's see if I can get this to work somehow...
- EnrolmentButtonCell.tsx is unused so that file could be removed

Reply:
Could be fixed yea! Since my winter holiday is starting, I would like to merge this now. Lets fix these in another PR.